package.json

```diff
@@ -89,12 +98,68 @@
       "*/**/test/unit/**/*.test.js"
     ]
   },
-  "standard": {
-    "env": [
-      "jest"
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js}": [
+      "npx prettier --write",
+      "npx eslint --fix"
+    ],
+    "*.{json,md,yml}": [
+      "npx prettier --write"
+    ]
+  },
+  "eslintConfig": {
+    "env": {
+      "node": true
+    },
+    "extends": [
+      "airbnb/base",
+      "plugin:jest/style",
+      "prettier"
+    ],
+    "ignorePatterns": [
+      "build/*"
     ],
-    "ignore": [
-      "test/**/*"
+    "parser": "@babel/eslint-parser",
+    "rules": {
+      "global-require": "off",
+      "no-console": "off",
+      "no-underscore-dangle": "off",
+      "no-use-before-define": [
+        "error",
+        {
+          "functions": false
+        }
+      ]
```

Comment:
- global-require: off
  - We use require a bit still for tests and GitLab. Once we update those we can remove this
- no-console
  - Console use is reasonable for an express app
- no-underscore-dangle
  - We use underscores to indicate private functions or vars
- no-use-before-define
  - Using functions before they're defined allows files to flow with highest level of abstraction at the top